### ISSUE TITLE
Can install via SPM in XCODE 15.1

### ISSUE THREAD

**Issue by KTDevelopment** [firebase/firebase-ios-sdk#12222]
### Description

I tried to install Firebase SDK via SPM following https://github.com/firebase/firebase-ios-sdk/blob/main/SwiftPackageManager.md

The project is an KMM Multiplatform project.

When installing via "Add Package Dependencies..." it's not possible to select any specific package.
After installation, packages cannot be resolved.

I'm quite sure I followed the instructions correctly.

What can I do to add firebase to my project?

### Firebase SDK Version

10.19.1

### Xcode Version

15.1

### Installation Method

Swift Package Manager

### Firebase Product(s)

Firestore

### Targeted Platforms

iOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

{
  "pins" : [
    {
      "identity" : "abseil-cpp-binary",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/abseil-cpp-binary.git",
      "state" : {
        "revision" : "bfc0b6f81adc06ce5121eb23f628473638d67c5c",
        "version" : "1.2022062300.0"
      }
    },
    {
      "identity" : "app-check",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/app-check.git",
      "state" : {
        "revision" : "5746b2d35c91c50581590ed97abe4c06b5037274",
        "version" : "10.18.0"
      }
    },
    {
      "identity" : "firebase-ios-sdk",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/firebase/firebase-ios-sdk",
      "state" : {
        "revision" : "c60c958e707c50a9cf8bcb7cfd7d51c566d726c5",
        "version" : "10.19.1"
      }
    },
    {
      "identity" : "googleappmeasurement",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/GoogleAppMeasurement.git",
      "state" : {
        "revision" : "6b332152355c372ace9966d8ee76ed191f97025e",
        "version" : "10.17.0"
      }
    },
    {
      "identity" : "googledatatransport",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/GoogleDataTransport.git",
      "state" : {
        "revision" : "a732a4b47f59e4f725a2ea10f0c77e93a7131117",
        "version" : "9.3.0"
      }
    },
    {
      "identity" : "googleutilities",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/GoogleUtilities.git",
      "state" : {
        "revision" : "bc27fad73504f3d4af235de451f02ee22586ebd3",
        "version" : "7.12.1"
      }
    },
    {
      "identity" : "grpc-binary",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/grpc-binary.git",
      "state" : {
        "revision" : "a673bc2937fbe886dd1f99c401b01b6d977a9c98",
        "version" : "1.49.1"
      }
    },
    {
      "identity" : "gtm-session-fetcher",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/gtm-session-fetcher.git",
      "state" : {
        "revision" : "115f75e43851774934d695449a4836123c3246e1",
        "version" : "3.2.0"
      }
    },
    {
      "identity" : "interop-ios-for-google-sdks",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/interop-ios-for-google-sdks.git",
      "state" : {
        "revision" : "2d12673670417654f08f5f90fdd62926dc3a2648",
        "version" : "100.0.0"
      }
    },
    {
      "identity" : "leveldb",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/firebase/leveldb.git",
      "state" : {
        "revision" : "9d108e9112aa1d65ce508facf804674546116d9c",
        "version" : "1.22.3"
      }
    },
    {
      "identity" : "nanopb",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/firebase/nanopb.git",
      "state" : {
        "revision" : "819d0a2173aff699fb8c364b6fb906f7cdb1a692",
        "version" : "2.30909.0"
      }
    },
    {
      "identity" : "promises",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/google/promises.git",
      "state" : {
        "revision" : "e70e889c0196c76d22759eb50d6a0270ca9f1d9e",
        "version" : "2.3.1"
      }
    },
    {
      "identity" : "swift-protobuf",
      "kind" : "remoteSourceControl",
      "location" : "https://github.com/apple/swift-protobuf.git",
      "state" : {
        "revision" : "65e8f29b2d63c4e38e736b25c27b83e012159be8",
        "version" : "1.25.2"
      }
    }
  ],
  "version" : 2
}


```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by ncooke3** [comment]
> After installation, packages cannot be resolved.

Hi @KTDevelopment, what is the error that showing when the package fails to resolve?

**Comment by KTDevelopment** [comment]
Here is a screenshot of the error
<img width="763" alt="Bildschirmfoto 2023-12-28 um 11 46 52" src="https://github.com/firebase/firebase-ios-sdk/assets/16954934/900ccb28-48c1-4768-bbc8-1098bc46b443">

**Comment by ncooke3** [comment]
Hi @KTDevelopment, try with the following change:

```diff
- import Firebase
+ import FirebaseCore
```

**Comment by KTDevelopment** [comment]
sadly this doesn't work
<img width="760" alt="Bildschirmfoto 2023-12-28 um 16 10 02" src="https://github.com/firebase/firebase-ios-sdk/assets/16954934/939545d8-1abc-4992-9963-35c1f348abb2">

**Comment by ncooke3** [comment]
Do you see Firebase listed as a dependency in the project navigator (left hand side of Xcode window)?
<img width="445" alt="Screenshot 2023-12-28 at 10 12 21 AM" src="https://github.com/firebase/firebase-ios-sdk/assets/36927374/3c40ff78-a3e5-44ad-9385-7c9062ea4a5e">

**Comment by ncooke3** [comment]
Alright, and if you open your target's settings, go to the `General` tab, and scroll down to `Frameworks, Libraries, and Embedded Content`, do you see Firebase SDKs have been added? 

Example: 
<img width="766" alt="Screenshot 2023-12-28 at 10 28 26 AM" src="https://github.com/firebase/firebase-ios-sdk/assets/36927374/6508fea2-c5f7-4e78-9ebc-e5edd297cd69">

**Comment by zafar-apk** [comment]
![image](https://github.com/firebase/firebase-ios-sdk/assets/78132342/f1776d06-c7b1-41c2-ae3b-b4819cf79303)
Having the same issue! But can't see the Framework, Libraries, and Embedded Content block. There's only Embedded Content block

**Comment by ncooke3** [comment]
Hi @zafar-apk, what version of Xcode are you using?

**Comment by zafar-apk** [comment]
Hi! 
same as @KTDevelopment 's Version 15.1

**Comment by KTDevelopment** [comment]
same here in Xcode version 15.1 there is no "Frameworks, Libraries, and Embedded Content" Block

**Comment by zafar-apk** [comment]
Here's how i fixed my issue
I noticed that i don't have a "Link Binary With Library" section in my target's build phase. So i added it.
![image](https://github.com/firebase/firebase-ios-sdk/assets/78132342/a77f3cf6-4833-4584-9ec9-bd68de03ac4e)
Finally i was able to see Framework, Libraries, and Embedded Content section. But couldn't add Firebase dependencies there. After File -> Packages -> Reset package caches all the dependencies were added to Framework, Libraries, and Embedded Content and i am able to work with Firebase now!
